dist/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>leaderboard</title>
+<script defer src="main.js"></script></head>
+
+<body>
+  <main id="lb-main">
+    <h1>Leaderboard</h1>
+    <div class="structure">
+
+      <section class="scores-section">
+        <div class="recent-scores">
+          <h2>Recent Scores</h2>
+          <button type="submit">Refresh</button>
+        </div>
+          <ul class="table">
+            <li class="name-scores">Name: 100</li>
+            <li class="name-scores">Name: 20</li>
+            <li class="name-scores">Name: 50</li>
```

Comment:
- [ ] You did great so far. But index.html main should contain only root element. The elements are supposed to be constructed from js DOM elements. Kindly make this change. :pray:

Reply:
Hello @henatan99. Please nothing in the requirement talked about js DOM. this is just a simple Setup plus Html markup. the line below is what the requirement is talking about.
"Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. Use this wireframe as a guide"

Reply:
Dear @OLIPLICHE , the students' project requirement documentation says as you quoted, but the reviewer requirement asks that the index.html main element should have only root element. But, I think your question is understandable and I will approve it now.

Reply:
Thank you